app/Resources/views/pagelayout_footer.html.twig

```diff
@@ -26,7 +26,7 @@
             <div class="row">
                 <div class="col-xs-offset-1 col-xs-5 space-top-half-more space-bottom-less">
                     <ul>
-                        <li class="white small-extra">Copyright &copy; 2015 eZ Systems AS (except where otherwise noted). All rights reserved. <a href="//ez.no/privacy-policy">Privacy Policy.</a> Powered by the <a href="//ez.no/ezpublish"> eZ Platform</a></li>
+                        <li class="white small-extra">Copyright &copy; 2016 eZ Systems AS (except where otherwise noted). All rights reserved. <a href="//ez.no/privacy-policy">Privacy Policy.</a> Powered by <a href="https://ezplatform.com/">eZ Platform</a></li>
```

Comment:
@DavidLiedle I already fix that in this PR https://github.com/ezsystems/ezplatform-demo/pull/47